queue_services/entity-emailer/requirements.txt

```diff
@@ -79,6 +79,6 @@ Werkzeug==1.0.1
 yarl==1.8.2
 zipp==3.15.0
 git+https://github.com/bcgov/business-schemas.git@2.18.33#egg=registry_schemas
-git+https://github.com/bcgov/lear.git#egg=legal_api&subdirectory=legal-api
+git+https://github.com/bcgov/lear.git@release-2.141.0#egg=legal_api&subdirectory=legal-api
```

Comment:
Updated this to pull in dependencies from release branch. Unit tests were failing because of DB version merge in main. Will have to revert it back to use main once we switch back to main

Reply:
Done